src/main/java/com/prgms/VoucherApp/domain/customer/controller/CustomerManagementController.java

```diff
@@ -1,30 +1,99 @@
+        this.input = input;
         this.output = output;
     }
 
+    @Override
+    public void run() {
+        boolean isRunning = true;
+        while (isRunning) {
+            output.printCustomerCommand();
+            Integer inputCustomerNumber = input.inputCustomerCommand();
+            CustomerCommand customerCommand = CustomerCommand.findByCustomerTypeNumber(inputCustomerNumber);
 
-    public void readBlackList() {
-        try {
-            List<CustomerDto> customerDtos = blackListReader.readBlackLists();
-            output.printBlackLists(customerDtos);
-        } catch (RuntimeException e) {
-            output.printErrorMsg(e.getMessage());
+            if (customerCommand.isCreate()) {
+                String inputCustomerStatus = input.inputCustomerStatus();
+                CustomerStatus inputStatus = CustomerStatus.findByStatus(inputCustomerStatus);
+                CustomerCreateReqDto customerCreateReqDto = new CustomerCreateReqDto(inputStatus);
+                customerDaoHandler.save(customerCreateReqDto);
+                continue;
+            }
+
+            if (customerCommand.isFindAll()) {
+                CustomersResDto findCustomers = customerDaoHandler.findAll();
+                output.printCustomers(findCustomers);
+                continue;
+            }
+
+            if (customerCommand.isFindOne()) {
+                String inputUUID = input.inputUUID();
+                UUID customerId = UUID.fromString(inputUUID);
+                customerDaoHandler.findOne(customerId)
+                    .ifPresentOrElse(output::printCustomer, output::printFindEmpty);
+                continue;
+            }
+
+            if (customerCommand.isFindByStatus()) {
+                String inputCustomerStatus = input.inputCustomerStatus();
+                CustomerStatus customerStatus = CustomerStatus.findByStatus(inputCustomerStatus);
+                CustomerCreateReqDto customerCreateReqDto = new CustomerCreateReqDto(customerStatus);
+                CustomersResDto findCustomers = customerDaoHandler.findByStatus(customerCreateReqDto);
+                output.printCustomers(findCustomers);
+                continue;
+            }
+
+            if (customerCommand.isFindBlackList()) {
+                CustomersResDto blackLists = customerDaoHandler.readBlackLists();
+                output.printBlackLists(blackLists);
+                continue;
+            }
+
+            if (customerCommand.isUpdate()) {
+                String inputUUID = input.inputUUID();
+                UUID customerId = UUID.fromString(inputUUID);
+                String inputCustomerStatus = input.inputCustomerStatus();
+                CustomerStatus customerStatus = CustomerStatus.findByStatus(inputCustomerStatus);
+                CustomerUpdateReqDto customerUpdateReqDto = new CustomerUpdateReqDto(customerId, customerStatus);
+                customerDaoHandler.update(customerUpdateReqDto);
```

Comment:
코드사이에 개행을 넣으면 가독성을 챙길 수 있습니다~ 지금은 읽기 좀 어렵네요🥲

```suggestion
                String inputUUID = input.inputUUID();
                UUID customerId = UUID.fromString(inputUUID);
                String inputCustomerStatus = input.inputCustomerStatus();
                
                CustomerStatus customerStatus = CustomerStatus.findByStatus(inputCustomerStatus);
                CustomerUpdateReqDto customerUpdateReqDto = new CustomerUpdateReqDto(customerId, customerStatus);
                
                customerDaoHandler.update(customerUpdateReqDto);
```